pkg/plugins/policies/meshhttproute/plugin/v1alpha1/gateway_routes.go

```diff
@@ -136,7 +136,7 @@ func sortRulesToHosts(
 	return meshroute.SortByHostname(hostInfosByHostname)
 }
 
-func GenerateEnvoyRouteEntries(host plugin_gateway.GatewayHost, toRules []ruleByHostname) []route.Entry {
+func generateEnvoyRouteEntries(host plugin_gateway.GatewayHost, toRules []ruleByHostname) []route.Entry {
```

Comment:
could it have been called from a parent project? if not then 👍 to making it private

Reply:
It's not